scripts/workers/code_upload_submission_worker.py

```diff
@@ -67,6 +115,40 @@ def get_submission_meta_update_curl(submission_pk):
     return curl_request
 
 
+def get_job_object(submission_pk, spec):
+    """Function to instantiate the AWS EKS Job object
+
+    Arguments:
+        submission_pk {[int]} -- Submission id
+        spec {[V1JobSpec]} -- Specification of deployment of job
+
+    Returns:
+        [AWS EKS Job class object] -- AWS EKS Job class object
+    """
+
+    job = client.V1Job(
+        api_version="batch/v1",
+        kind="Job",
+        metadata=client.V1ObjectMeta(
+            name="submission-{0}".format(submission_pk)
+        ),
+        spec=spec,
+    )
+    return job
+
+
+def get_init_container(submission_pk):
+    curl_request = get_submission_meta_update_curl(submission_pk)
+    # Configure init container
+    init_container = client.V1Container(
+        name="init-container",
+        image="busybox",
+        command="/bin/sh",
+        args=["-c", curl_request],
```

Comment:
@savish28 this would need to be rebased. See the changes in my recent [PR](https://github.com/Cloud-CV/EvalAI/pull/3497). `busybox` doesn't have `curl` so we have to use a ubuntu image and install curl explicitly. We will merge my PR today, you can rebase after that.

Reply:
@Ram81, Sure would do the required changes as soon as the respective PR is merged.